dev/com.ibm.ws.transport.http/src/com/ibm/ws/http/netty/pipeline/CRLFValidationHandler.java

```diff
@@ -20,7 +20,7 @@
 
 public class CRLFValidationHandler extends ChannelInboundHandlerAdapter {
```

Comment:
We should add some documentation to this class to explain why this is needed with Netty to match Legacy behavior